dash/test/algorithm/SortTest.cc

```diff
@@ -29,6 +29,8 @@ TEST_F(SortTest, ArraySort)
 
   auto const true_sum = dash::accumulate(array.begin(), array.end(), 0);
 
+  int wait = 0;
+  while(wait);
```

Comment:
"OK, I'll wait." Said no CI ever...

Reply:
Well that is a very important code snippet as you can imagine. ;)

I'm deleting it. And will also address you change requests. :) Thanks for your review.